.gitignore

```diff
@@ -122,3 +122,6 @@ TEST-**.xml
 
 # Mac OS .DS_Store, which is a file that stores custom attributes of its containing folder
 .DS_Store
+
+# VS Code dev containers extension
+.devcontainer
```

Comment:
I'd rather not have this change, instead feel free to add devcontainer configuration. 

You can use `echo '.devcontainer' >> .git/info/exclude` if you need to ignore it locally.

Reply:
https://github.com/onerandomusername/monty-python/pull/233/commits/8c12e183a48586306763de7bea0f48cad6225164